docs/installing.md

```diff
@@ -134,6 +134,24 @@ All the arguments are optional and most users should not need them:
 - `requirements_file`: Path to the requirements.yml file to install role and
   collection dependencies.
 
+In order to install roles and collections from private repositories, you can
+create an
+[access token](https://docs.github.com/en/authentication/keeping-your-account-and-data-secure/managing-your-personal-access-tokens#about-personal-access-tokens),
```

Comment:
The steps might be a bit clearer if they were separated into a numbered list:
1. Create an access token
2. Add access token as a deploy secret
3. Add the following step before the ansible-lint step

Reply:
Hi, thank you for the input. I have updated the PR with the changes.